docs/openapi/api-internal-pn-bff-pa-info.yaml

```diff
@@ -93,4 +93,51 @@ paths:
           content:
             application/problem+json:
               schema:
-                $ref: 'common-refs.yaml#/components/schemas/Problem'
\ No newline at end of file
+                $ref: 'common-refs.yaml#/components/schemas/Problem'
+  
+  '/bff/v1/groups':
+    get:
+      summary: Get groups
+      description: |-
+        This operation retrieves the groups from Self Care.
+      operationId: getGroupsV1
+      #      security:                                      # ONLY EXTERNAL
+      #      - bearerAuth: [ ]                            # ONLY EXTERNAL
+      tags:
+      - InfoPa
+      parameters: # NO EXTERNAL
+      - $ref: 'common-refs.yaml#/components/parameters/uidAuthFleet'                          # NO EXTERNAL
+      - $ref: 'common-refs.yaml#/components/parameters/cxIdAuthFleet'                         # NO EXTERNAL
+      - $ref: 'common-refs.yaml#/components/parameters/cxGroupsAuthFleet'                     # NO EXTERNAL
+      - name: statusFilter
+        description: Se valorizzato indica di tornare solo i gruppi nello stato passato
+        in: query
+        required: false
+        schema:
+          $ref: '#/components/schemas/BffPaGroupStatus'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BffPaGroupList'
+        '400':
+          description: Invalid input
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'common-refs.yaml#/components/schemas/Problem'
+        '500':
+          description: Internal Server Error
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'common-refs.yaml#/components/schemas/Problem'
+
+components:
+  schemas:
+    BffPaGroupStatus:
```

Comment:
Move this and the next schema to the file where there are the schemas for institution and products.

Rename the institution-and-product-schemas dir and institution-and-product.yaml file with info-pa-schemas and info-pa.yaml